cards/src/main/resources/cards/custom/doombubbles/mage/spell_rebellious_flame.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Rebellious Flame",
```

Comment:
It's not good to have multiple cards with the same name, but I think this will do.